packages/project/src/shared/styles/style-rules.test.ts

```diff
@@ -0,0 +1,109 @@
+import { test, expect } from "@jest/globals";
+import type { Styles } from "@webstudio-is/react-sdk";
+import { getStyleRules, getPresetStyleRules } from "./style-rules";
+
+test("get a list of style rules grouped by instance and breakpoint", () => {
```

Comment:
no describe? I thought we should always have a describe for output

Reply:
No, it's mocha requirement. jest allows to wrap with only "test"

Reply:
I mean technically its allowed, but I thought it would result in bad reporting, without proper encapsulation

Reply:
If file name is descriptive enough we may not need to wrap all tests.

```
style-rules.test.ts
  get a list of style rules grouped by instance and breakpoint
    getStyleRules
  something else
    getPresetRules
```

```
style-rules.test.ts
  get a list of style rules grouped by instance and breakpoint
  something else
```